src/azure-cli/azure/cli/command_modules/monitor/_help.py

```diff
@@ -1221,7 +1221,7 @@
 type: command
 short-summary: Create some linked storage accounts for log analytics workspace.
 examples:
-  - name: Create two linked storage accounts for a log analytics workspace
+  - name: Create two linked storage accounts for a log analytics workspace using the name of the storage account.
```

Comment:
Could we add an example that uses the resource id of the storage account?
Also please use full name for storage account instead of abbreviation (SA1, SA2) to be more clear.

Reply:
Done